week1/homework/test/server.test.js

```diff
@@ -5,7 +5,7 @@
 * purpose of this project
 */
 const request = require('supertest');
-const test    = require('ava');
+const test = require('ava');
```

Comment:
Your linter and code editor are probably responsible for the changes made to this file, but you should not have committed it. Also note that it's important to respect the style of the code you're contributing to.

Reply:
Hi Alex, I believe I must have done the steps not correctly. I am now on the process of correcting issues, therefore I will start the homework again. I will watch closely the paths given to avoid any errors and I will take note of your feed back.

Thank you.